### ISSUE TITLE
Move custom shadows for experimental out of main repo

### ISSUE THREAD

**Issue by xian** [robolectric/robolectric#4504]

**Comment by MGaetan89** [comment]
Is there something that needs to be done for this ticket, or can it be closed? cc @hoisie @brettchabot

**Comment by hoisie** [comment]
Yeah it can be closed. I think for some version of Android (I think Q), Robolectric supported the prerelease version. However, supporting prerelease versions is not really feasible, due to how frequently it changes.
